.github/workflows/ci.yml

```diff
@@ -56,3 +56,29 @@ jobs:
       - run: sudo apt-get update
       - run: sudo apt-get install -yq devscripts
       - run: make itest_${{ matrix.dist }}
+      - uses: actions/upload-artifact@v3
+
+        with:
+          name: deb-${{ matrix.dist }}
+          path: dist/paasta-tools_*.deb
+  cut_release:
+    runs-on: ubuntu-22.04
+    needs: build_debs
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir -p dist/bionic dist/jammy
```

Comment:
(non-blocking) hmm, i wonder if we can use a matrix for doing the ubuntu version stuff so that there's just a single place we need to update when we're supporting/dropping a given ubuntu version

Reply:
I don't know of a way to do this with softprops/action-gh-release (which uploads all release files during that one step). If I had a matrix for release, I'd end up with two release jobs, each trying to make their own release.

In a personal repo, I've used [actions/upload-release-artifact](https://github.com/actions/upload-release-asset) and [actions/create-release](https://github.com/actions/create-release) to create the release first, then upload the artifacts directly from the build steps, but create-release and upload-release-artifact are deprecated; their readmes point to softprops/action-gh-release.